app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(names: str) -> None:
+    command, old_file, new_file = names.split(" ")
```

Comment:
The variable names `old_file` and `new_file` are descriptive, but consider using a more descriptive name for `command`, such as `command_name` or `operation`, to improve clarity.